src/core/components/card/metrics-card-component.tsx

```diff
@@ -51,8 +51,8 @@ const MetricsCard: React.FC<MetricsCardProps> = ({
         )}
         {!isEmpty(outOfStockCount) && (
           <div className={styles.countGrid}>
-            <span className={styles.belowMinLabel}>{t('itemsBelowMin', 'Items Below Min')}</span>
-            <span className={styles.aboveMaxLabel}>{t('itemsAboveMax', 'Items Above Max')}</span>
+            <span className={styles.belowMinLabel}>{t('itemsUnderstocked', 'Items understocked')}</span>
+            <span className={styles.aboveMaxLabel}>{t('itemsOverstocked', 'Items overstocked')}</span>
```

Comment:
```suggestion
            <span className={styles.belowMinLabel}>{t('understockedItems', 'Understocked items')}</span>
            <span className={styles.aboveMaxLabel}>{t('overstockedItems', 'Overstocked items')}</span>
```

Maybe this reads better.

Reply:
Resolved